doxygen warning

```
/home/runner/work/AMICI/AMICI/include/amici/vector.h:240: warning: argument 's' of command @param is not found in the argument list of amici::AmiVector::boost::serialization::serialize(Archive &ar, AmiVector &v, unsigned int version)
```
see https://github.com/AMICI-dev/AMICI/actions/runs/15905879464/job/44860513194?pr=2838

--- a/include/amici/vector.h
+++ b/include/amici/vector.h
@@ -238,7 +238,7 @@ class AmiVector {
     /**
      * @brief Serialize AmiVector (see boost::serialization::serialize)
      * @param ar Archive to serialize to
-     * @param s Data to serialize
+     * @param v Data to serialize
      * @param version Version number
      */
     template <class Archive>

